tests/CMakeLists.txt

```diff
@@ -34,6 +38,9 @@ function(naive_tsearch_add_unittest NAME)
         $<IF:$<C_COMPILER_ID:MSVC>,/W4,-Wall -Wextra>)
     set_target_properties(${NAME} PROPERTIES CXX_STANDARD 11)
     add_test(NAME "test_${NAME}" COMMAND ${NAME})
+    if(MSVC)
+        target_compile_options(${NAME} PRIVATE /EHsc)
```

Comment:
Perhaps you could add a comment in the file to explain `/EHsc` and why it is necessary ?

I admit that I do not usually require comments for GCC flags, because I am familiar with the GCC flags, but also I think GCC flags are more self-explanatory than MSVC ones.

Reply:
The new comment looks good.